problems/1.types+operators.js

```diff
@@ -2,17 +2,33 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
+function isNumber(val) {
+ 
+  if(val !== " ") {
```

Comment:
We need to check of typeof val === "number" . 
Other strings are also not equal to " "